test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
In VS can you "type in" that namespace in a CS file, see what types are there, and then Right-Click, Go To Definition, and see what assembly those are coming from? This is very suspicious...